modules/exoplanets/exoplanetshelper.h

```diff
@@ -33,42 +33,33 @@ struct Exoplanet {
     float a;            // Orbital semi-major axis in AU
     double aUpper;      // Upper uncertainty of orbital semi-major axis
     double aLower;      // Lower uncertainty of orbital semi-major axis
-    double ua;          // Uncertainty of orbital semi-major axis
-    float bigOm;        // Longitude of ascending node in degrees
-    float bigOmUpper;   // Upper uncertainty of longitude of ascending node
-    float bigOmLower;   // Lower uncertainty of longitude of ascending node
-    float uBigOm;       // Uncertainty of longitude of ascending node
+    float bigOmega;     // Longitude of ascending node in degrees
+    float bigOmegaUpper;// Upper uncertainty of longitude of ascending node
```

Comment:
Nitpicky:  an extra space here would make it a bit clearer optically where the comments begin